build.sh

```diff
@@ -11,6 +11,7 @@ usage()
     echo "clean - optional argument to force a clean build."
     echo "verbose - optional argument to enable verbose build output."
     echo "clangx.y - optional argument to build using clang version x.y."
+    echo "installcli - should download CLI and not use cached copy of the CLI."
```

Comment:
Since CLI is also actively under development, if you have a cached copy of the CLI that is no longer valid when you issue build.cmd, then you can specify installcli. The other option is to manually remove the bin\tools. Would you prefer not having this option?